pom.xml

```diff
@@ -164,6 +164,24 @@
                     </execution>
                 </executions>
             </plugin>
+            <!--            Open Rewrite-->
+            <plugin>
+                <groupId>org.openrewrite.maven</groupId>
+                <artifactId>rewrite-maven-plugin</artifactId>
+                <version>5.21.0</version>
+                <configuration>
+                    <activeRecipes>
+                        <recipe>org.openrewrite.java.spring.boot3.UpgradeSpringBoot_3_1</recipe>
```

Comment:
@Shiroo77 if we're using the recipe `UpgradeSpringBoot_3_1` (and not `UpgradeSpringBoot_3_0`) then we should also rename the commit message and this PR accordingly